include/autocheck/generator.hpp

```diff
@@ -271,8 +271,12 @@ namespace autocheck {
       std::tuple<Gens...> gens;
 
     public:
-      cons_generator() :
-        gens(Gens()...) {}
+      cons_generator() 
+#ifndef _MSC_VER
+		  :
```

Comment:
To be clear, you might need to expand tabs in the other edits; I'm not going to tag each one.